app/services/update_user.rb

```diff
@@ -0,0 +1,14 @@
+class UpdateUser
```

Comment:
I will confess I am confused by this additional level of abstraction. Why not just call `user.update!(attributes)` in-place rather than having this small service class?